core/unit_test/TestMathematicalFunctions.hpp

```diff
@@ -1290,12 +1290,12 @@ struct TestAbsoluteValueFunction {
     if (abs(static_cast<KE::half_t>(4.f)) != static_cast<KE::half_t>(4.f) ||
         abs(static_cast<KE::half_t>(-4.f)) != static_cast<KE::half_t>(4.f)) {
       ++e;
-      KOKKOS_IMPL_DO_NOT_USE_PRINTF("failed abs(KE::half_t)\n");
+      Kokkos::printf("failed abs(KE::half_t)\n");
```

Comment:
I am generally fine with 22ea496674a2ac44d6eb77d519fa3b0b726a6688 but these changes do not seem to conflict with the rest of the PR and therefore probably should have been proposed in some other PR to avoid the noise.